benchmark/README.md

```diff
@@ -1,41 +1,99 @@
-Swift Benchmark Suite
-=====================
 
-This directory contains the Swift Benchmark Suite.
+# Swift Benchmark Suite
 
-Running Swift Benchmarks
-------------------------
+This directory contains the Swift Benchmark Suite. This is meant to be a suite
+of regression, micro-benchmark, small, and medium sized benchmarks that is used
+to validate the performance of the swift compiler and standard library.
 
-To run Swift benchmarks, pass the `--benchmark` flag to `build-script`. The
-current benchmark results will be compared to the previous run's results if
-available. Results for each benchmark run are logged for future comparison.
+## Building Swift Benchmarks
 
-For branch based development, take a baseline benchmark on the Swift `master`
-branch, switch to a development branch containing potentially performance
-impacting changes, and run the benchmarks again. Upon benchmark completion, the
-benchmark results for the development branch will be compared to the most
-recent benchmark results for `master`.
+There are 3 ways to build the benchmark suite today:
+
+1. In tree using build-script and cmake.
+2. Standalone using swiftpm.
+3. Standalone using cmake.
+
+**NOTE:** There are various level of support here. Specifically:
+
+1. Linux is only supported by the standalone cmake build.
+2. Non macOS Darwin platforms do not today support the swiftpm build.
+3. macOS supports all 3 ways of building.
+
+This is not due to to intention, but rather than more engineering time would be
+required to productize these builds.
 
-Building with build-script
---------------------------
+### In tree using build-script and cmake
 
-By default, Swift benchmarks for OS X are compiled during the Swift build
-process. To build Swift benchmarks for additional platforms, pass the following
-flags:
+By default, Swift benchmarks on mac OS are compiled in tree during the Swift
+build process, e.g.:
 
-    $ swift/utils/build-script --ios --watchos --tvos
+    $ swift/utils/build-script --release --no-assertions
+
+To build Swift benchmarks for additional platforms, pass the following flags:
+
+    $ swift/utils/build-script --release --no-assertions --ios --watchos --tvos
 
 OS X benchmark driver binaries are placed in `bin` alongside `swiftc`.
 Additional platform binaries are placed in the `benchmark/bin` build directory.
 
-Building Independently
-----------------------
+**NOTE:** We assume that the user has installed all necessary swift dependencies
+since we will build the rest of the compiler as well.
+
+### Building standalone with SwiftPM
+
+To build the benchmark suite with SwiftPM just:
+
+    $ cd ./swift/benchmark
+    $ swift build
+
+This can be used to generate an xcode project that an be used to edit the
+benchmarks:
+
+    $ cd ./swift/benchmark
+    $ swift package generate-xcodeproj
+    $ open swiftbench.xcodeproj
+
+**NOTE:** By default the xcode project's sheme for SwiftBench will build the
```

Comment:
"**NOTE:** By default, the xcode's project scheme for SwiftBench will build the"

Reply:
The apostrophe here is correct. I am not talking about xcode (the app)'s project scheme. I am talking about the "xcode project's" scheme. The spelling error is correct here.